nixos/tests/audit.nix

```diff
@@ -0,0 +1,24 @@
+{
+
+  name = "audit";
```

Comment:
Shouldn't this test be named `auditd` ?

Reply:
I named it audit because it tests both. Also made this more explicit in enabling both options explicitly.